hedgehog-example/src/Test/Example/Basic.hs

```diff
@@ -70,8 +70,17 @@ prop_foo =
 
     guard (y `mod` 2 == (1 :: Int))
 
-    assert $
-      y < 87 && x <= 'r'
+    diff y (<) 87
+    diff x (<=) 'r'
+
+    comparison y (<) 87
+    comparison x (<=) 'r'
+
+    relation y (<) 87
+    relation x (<=) 'r'
+
+    relate y (<) 87
+    relate x (<=) 'r'
```

Comment:
Would appreciate some comments on which name people like best or some other one? The main point of this is to provide a diff, otherwise you'd just use assert and annotate.

Also is the infix style crazy? it's kind of non-haskellish but it does read pretty nice.

Reply:
no strong opinions, but `comparison` is my favourite. I like the infix style, I think it makes the meaning pretty obvious.

Reply:
I like `diff` and `relate`, I like how they sound, *diff this with that*, *relate this with that*.

Reply:
One thing @moodmosaic and I discussed with `comparison` is that it's not a verb, like if we could use `compare` that would be awesome but it's already taken by `Ord`